docker/install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#  Copyright (C) 2018-2024 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# Build ros2
+cd ~/
+#source /home/carma/catkin/setup.bash
```

Comment:
This commented out line will probably be required for the carma-j2735 update, but can be removed for this feature branch